pandas/io/pytables.py

```diff
@@ -580,16 +580,37 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_value, traceback):
         self.close()
 
-    def keys(self) -> List[str]:
+    def keys(self, include: Optional[str] = "pandas") -> List[str]:
         """
         Return a list of keys corresponding to objects stored in HDFStore.
 
+        Parameters
+        ----------
+        include : str, default 'pandas'
+                When kind equals 'pandas' return pandas objects
+                When kind equals 'native' return native HDF5 Table objects
+                Otherwise fail with a ValueError
+
         Returns
         -------
         list
             List of ABSOLUTE path-names (e.g. have the leading '/').
+
+        Raises
+        ------
+        raises ValueError if kind has an illegal value
         """
-        return [n._v_pathname for n in self.groups()]
+        if include == "pandas":
+            return [n._v_pathname for n in self.groups()]
+
+        if include == "native":
```

Comment:
use elif

Reply:
Not really needed in this case as the previous if returns in all cases. Why would that help us?

Reply:
can you change this (its more idiomatic)

Reply:
fixed